### ISSUE TITLE
Console error

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/betarena-ad-widget-main#23]
index.mjs:447 Error: <svg> attribute viewBox: Expected number, “0 0 100% 100%“

![image](https://user-images.githubusercontent.com/37311649/129891624-1130228d-ad5f-4c7a-8a2d-0d07b1808633.png)

**Comment by migbash** [comment]
Yeah, this error can only be fixed by changing the `node_modules\svelte-content-loader\src\ContentLoader.svelte` file and **removing** the `viewBox` parameter completely, which does not take `% (percentage) values` and only allows for digits. This is ok and still makes the svg responsive and fluid in size as we are using `width=100%` and height=100%` anyways.
